packages/shared/src/components/auth/CodeVerificationForm.tsx

```diff
@@ -46,12 +46,12 @@ function CodeVerificationForm({
 
   const onCodeVerification = async (e) => {
     e.preventDefault();
-    trackEvent({
-      event_name: AuthEventNames.SubmitForgotPassword,
-    });
+    // trackEvent({
+    //   event_name: AuthEventNames.SubmitForgotPassword,
+    // });
```

Comment:
Just commenting to check if this is intended to be removed in prod

Reply:
Ah good point actually needed to check with @idoshamun if we wanted specific tracking?
(this above is just a copy of existing tracking)

Reply:
Yes let's add 
<img width="550" alt="image" src="https://github.com/dailydotdev/apps/assets/1993245/e70e750a-a65d-456d-88b9-78f34f4e0f61">